netlify-functions-ecommerce/.env.example

```diff
@@ -0,0 +1,27 @@
+#Environment to choose. Possible values: astra, jsonapi
```

Comment:
I would advise against using non-standard values for NODE_ENV. So best to not rely on anything other than "local", "development", or "production" for NODE_ENV.

Better to just have an IS_ASTRA environment variable.